create.sql

```diff
@@ -557,7 +571,8 @@ CREATE TABLE step_f2 (
     exposure_time integer NOT NULL,
     filter identifier NOT NULL,
     lyot_wheel identifier NOT NULL,
-    disperser identifier NOT NULL
+    disperser identifier NOT NULL,
+    window_cover identifier NOT NULL
```

Comment:
The name `step_f2` was a little confusing at first.  There is no F2 step but an F2 configuration is part of every kind of step: science, gcal, bias, etc.  That is what we have here, and actually I can't think of a better name.